analyzers/tests/SonarAnalyzer.UnitTest/TestCases/SymbolicExecution/Roslyn/ObjectsShouldNotBeDisposedMoreThanOnce.CSharp8.cs

```diff
@@ -0,0 +1,113 @@
+﻿using System;
+using System.IO;
+
+namespace Tests.Diagnostics
+{
+    namespace CSharp8
+    {
```

Comment:
This is just scaffolding noise that makes the test cases harder to read and maintain. Without a practical impact - should be removed
```suggestion
```